test/utils.test.ts

```diff
@@ -1028,7 +1027,7 @@ describe("utils", () => {
         it('correctly records all changed extension settings', async () => {
             const reporter = createFakeTelemetryReporter();
             utils.reportExtensionSettings(reporter);
-            expect(reporter.sendTelemetryEvent).toBeCalledWith('user/settingsChangedAtLaunch', { welcome: 'true', enableNetwork: 'false' });
+            expect(reporter.sendTelemetryEvent).toBeCalledWith('user/settingsChangedAtLaunch', { enableNetwork: 'false' });

```

Comment:
Since we have a patch that removes `enableNetwork` we should choose something else to use in test code.

Reply:
Agreed. In fact, I switched this to use `isHeadless` in the `enableNetwork` patch already :)
![image](https://user-images.githubusercontent.com/785009/167667621-7bf3bf3f-749c-4c0f-ae62-e46eb74669e3.png)